packages/upswyng-server/src/routes/api/resources/next-schedule.ts

```diff
@@ -0,0 +1,135 @@
+import { ResourceSchedule, TScheduleItem } from "@upswyng/upswyng-core";
```

Comment:
Turns out that this endpoint is necessary. I was able to handle the RRule performance issues by using a combo of limiting [`.all()`](https://github.com/jakubroztocil/rrule#rruleprototypealliterator) and using [`.after()`](https://github.com/jakubroztocil/rrule#rruleprototypeafterdt-incfalse). Discovered that by reading the documentation... imagine that.

Reply:
I will remove this before merging this PR, but I think this and the the cache may be helpful to reference for #274

Reply:
Well yes, this is quite helpful 😄

Reply:
@jollyjerr I deleted this endpoint but will keep the cache so you can use it in yours 👍